db/migrate/20191030125101_normalize_phone_number.rb

```diff
@@ -0,0 +1,33 @@
+class NormalizePhoneNumber < ActiveRecord::Migration[5.2]
+  def up
+    Member.find_each do |m|
+      phone_number = normalize(m.phone_number)
+      emergency_phone_number = normalize(m.emergency_phone_number)
+
+      m.update_attribute :phone_number, phone_number
+      m.update_attribute :emergency_phone_number, emergency_phone_number
+    end
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
+  end
+
+  # normalize and validate phone number
```

Comment:
This normalization logic probably isn't enough: we have a lot of phone numbers that are landlines, and some German and other odd values as well. I think that it's not a problem if we lose a member's phone number if it really is invalid, but we should in that case at least log the original value and which member the number belonged to (so the board can e.g. mail them if the member is still active).

Please try to handle Dutch local landlines: (`0[^0]\d{8}`, replace leading `0` with `+31`) which should also get the Dutch cellphones

Reply:
Landlines should now be handled.

Invalid numbers are added to the member's comments so we don't lose them.